source/hardware/controllers/numark_n4.rst

```diff
@@ -79,9 +79,7 @@ Hotcues
 ^^^^^^^
 
 The four lower buttons of the topContainer are hotcues. They can be changed by using the corresponding Encoder [HOTCUE PAGE]. The current assignment will then be shown for one second in the upper row.
-Currently up to 4 Pages are supported (with 4 hotcues each). A demo of how this looks can be seen on `JsFiddle <https://jsfiddle.net/Swiftb0y/q9zbgvxj/1/>`__ (not done yet) (the slider represents the
-Encoder).
-
+Currently up to 4 Pages are supported (with 4 hotcues each).
```

Comment:
```suggestion
Currently up to 4 Pages are supported (with 4 hotcues each).

```

You need to keep the linebreak. 
![Screenshot from 2023-02-04 21-43-06](https://user-images.githubusercontent.com/12380386/216788831-21bd68ca-f117-4f3b-aba2-fc0d401cd8d3.png)